src/activities/PartialTranslation.tsx

```
@@ -0,0 +1,161 @@
+        .replace(/[^a-zA-Z\u0D00-\u0D7F]+/g, "");
+
+    const entries = Lexicon.getTranslation(clean);
+
+    if (!entries) return <>{word}</>;
+
+    return (
+        <Dictionary
+            content={
+                <div>
+                    {entries.map(([mal, tr], i) => (
+                        <div className="definition-container">
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)